markets/perps-market/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@synthetixio/perps-market",
-  "version": "3.3.20",
+  "version": "3.3.17",
```

Comment:
downgrading?

Reply:
this PR is not where we should be upgrading; its when we deploy.  i think it was bumped up by mistake here